os/src/main/kotlin/com/gojuno/commander/os/Processes.kt

```diff
@@ -50,15 +50,15 @@ fun process(
 
             val command: List<String>
```

Comment:
Seems like it is possible to make it like this:

```kotlin
val command: List<String> = if (unbufferedOutput) {
    commandAndArgs
} else {
    when (os()) {
        Linux -> listOf()
        Mac -> listOf()
        Windows -> commandAndArgs
    }
}
```

Reply:
Ahh, sure